MoltenVK/MoltenVK/GPUObjects/MVKPixelFormats.mm

```diff
@@ -1238,8 +1238,6 @@
 	addMTLPixelFormatCapabilities( tvOS_GPUFamily1_v2, BGRA8Unorm_sRGB, All );
 	addMTLPixelFormatCapabilities( tvOS_GPUFamily2_v1, BGRA8Unorm_sRGB, All );
 
-	addMTLPixelFormatCapabilities( tvOS_GPUFamily2_v1, RGB10A2Unorm, All );
-	addMTLPixelFormatCapabilities( tvOS_GPUFamily2_v1, RGB10A2Uint, RWCM );
```

Comment:
What are you trying to accomplish here?

`addMTLPixelFormatCapabilities()` adds capabilities to the default capabilities. This will leave `RGB10A2Unorm` with `RFCMRB` (everything but `Write` capabilities), as set in `initMTLPixelFormatCapabilities()`.

If you really mean that the formats are not available at all in tvOS (which seems strange that they are defined for the platform), then use `disableAllMTLPixelFormatCapabilities()`. 

If you just mean that the format can't be used for surfaces in `CAMetalLayer`, then `getSurfaceFormats()` is the correct place to fix that (probably by wrapping it being set there in a `#if MVK_MACOS` conditional compile macro.

Reply:
I see okay, thanks for the clarification. This is my first time using Metal or Vulkan, so I'm trying to get up to speed with all these concepts as quickly as possible.

I will revert the last commit.